app/code/Transoft/Blog/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <section id="catalog" translate="label" type="text" sortOrder="40" showInDefault="1" showInWebsite="1" showInStore="1">
+        <class>separator-top</class>
+        <label>Catalog</label>
+        <tab>catalog</tab>
+        <resource>Magento_Catalog::config_catalog</resource>
+        <group id="blog" translate="label" type="text" sortOrder="90" showInDefault="1" showInWebsite="1" showInStore="1">
+            <label>Blog Configuration</label>
+            <field id="blog_applied_to" translate="label" type="multiselect" sortOrder="1" showInDefault="1" showInWebsite="0" showInStore="0">
```

Comment:
Please add field comment.

Reply:
fixed

Reply:
I don't see changes

Reply:
fixed

Reply:
Why did you remove field **comment**. Please refert changes and add comment translation

Reply:
not sure why this happens, now fixed

Reply:
> Why did you remove field comment. Please refert changes and add comment translation
I can't see that comment is back